tvb_contrib/tvb/contrib/cosimulation/cosimulator.py

```diff
@@ -78,6 +78,27 @@ class CoSimulator(Simulator):
     _cosim_monitors_noncoupling_indices = []
     _cosim_monitors_coupling_indices = []
 
+    @property
```

Comment:
Using "@property" cannot be efficient in some situations because it is computed each time it is called.
For the moment, there are few calls to it, so it should be fine.